integreat_cms/cms/templates/analytics/__page_hix_row.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n %}
+{% load rules %}
+{% load content_filters %}
+{% load page_filters %}
+{% with page_translation.page.id as page_id %}
+    {% with page_translation.hix_score as hix %}
+        <tr class="{% if hix < hix_threshold %}bg-red-300{% endif %}">
+            <td>
+                <a href="{% url 'edit_page' region_slug=request.region.slug language_slug="de" page_id=page_id %}">
```

Comment:
I'm a bit torn here, and I think we need @hauf-toni 's opinion on this. 
So at the moment in order to improve the HIX value of your page you should click on the page title. I would argue that it would be better to have a separate link at the end of the table, that makes the call to action a bit clearer and says something like "Click here to improve HIX value". But I don't think I'm qualified to decide this

Reply:
I think the text "Click here to improve HIX value" is a bit long if it is duplicated for all pages...
Also, Users are already used to click on the page title to get to the edit form.
Maybe it would be sufficient to add a hover effect with a grey background on the hovered table row similar to the page tree?